File: TestAutomation/Test/LoginPage.cs
using System;
using System.Text;
using System.Text.RegularExpressions;
using System.Threading;
using NUnit.Framework;
using OpenQA.Selenium;
using OpenQA.Selenium.Firefox;
using OpenQA.Selenium.Support.UI;
using OpenQA.Selenium.Chrome;

namespace TestAutomation.Test
{
    internal class LoginPage : Page
    {

        public LoginPage(IWebDriver driver) : base(driver)
        {

        }
        internal void Open()
        {
            driver.Navigate().GoToUrl(baseURL + "wp-admin");
        }
        internal MainAdminPage LogIn()
        {
            InsertText(By.Id("user_login"), "szkolenieautomatyzacja");
            InsertText(By.Id("user_pass"), "QW12qw12");
            Click(By.Id("wp-submit"));
            return new MainAdminPage(driver);
        }


    }
}

File: src/test/java/rst/wordpress/LoginPage.java
package rst.wordpress;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;

/**
 * Created by Administrator on 2016-09-23.
 */
public class LoginPage extends Page{

    public LoginPage(WebDriver driver) {
        super(driver);
    }

    public AdminMainPage logIn(String user, String password) {

        driver.get(baseUrl + "/");
        driver.get(baseUrl + "/wp-admin");

        insertText(By.id("user_login"), "szkolenieautomatyzacja");
        insertText(By.id("user_pass"), "QW12qw12");
        driver.findElement(By.id("wp-submit")).click();


        return new AdminMainPage(driver);
    }

}
